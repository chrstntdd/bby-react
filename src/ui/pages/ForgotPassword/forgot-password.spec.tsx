import React from 'react';
import { shallow } from 'enzyme';

import { ForgotPassword } from './';

describe('DUMB <ForgotPassword/>', () => {
  const props = {
    handleSubmit: jest.fn()
  };
  it('should render without crashing', () => {
    const wrapper = shallow(<ForgotPassword {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
