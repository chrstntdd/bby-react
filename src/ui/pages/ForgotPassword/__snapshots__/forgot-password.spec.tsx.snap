// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DUMB <ForgotPassword/> should render without crashing 1`] = `
<section
  id="forgot-password-wrapper"
>
  <LoadingIndicator
    message="One moment please."
    waiting={false}
  />
  <div
    className="show"
    id="forgot-password-card"
  >
    <form
      aria-labelledby="forgot-password"
      onSubmit={[Function]}
    >
      <fieldset>
        <legend
          id="forgot-password"
        >
          Forgot password
        </legend>
        <div
          className="input-wrapper"
        >
          <Input
            id="employeeNumberInput"
            inputRef={
              Object {
                "current": null,
              }
            }
            label="Employee Number"
            type="text"
          />
          <span
            className="focus-border"
          />
        </div>
        <button
          type="submit"
        >
          Reset Password
        </button>
      </fieldset>
    </form>
  </div>
</section>
`;
